test/ParserTest.php

```diff
@@ -119,4 +119,22 @@ public function testCancelReturnsInternalBuffer() {
 
         $this->assertSame("d", $parser->cancel());
     }
+
+    public function testIsValidShouldReturnTrue() {
```

Comment:
I think the test name should rather be something like `testIsValidOnValidParser`, it should say _when_ `isValid` should return `true`.

Reply:
I tihnk this test method name has been changed and it seems that checking the ```isValid``` return true on the non finished parser.
So this method name should be changed into the ```testIsValidOnNonFinishedParser```?